modules/hocon/src/main/scala/ciris/hocon/package.scala

```diff
@@ -0,0 +1,33 @@
+package ciris
+
+import cats.Show
+import ciris.{ConfigException => _, _}
+import com.typesafe.{config => tsc}
+import cats.syntax.all
+
+import scala.util.Try
+
+package object hocon extends HoconConfigDecoders {
+
+  final class HoconAt(configV: ConfigValue[tsc.Config]) {
```

Comment:
My preference would be to have this as a `sealed abstract class`, similarly to [ConfigMapInNamespace](https://github.com/ovotech/ciris-kubernetes/blob/master/src/main/scala/ciris/ConfigMapInNamespace.scala) in ciris-kubernetes.